src/test/java/org/mockbukkit/mockbukkit/util/UnsafeValuesTest.java

```diff
@@ -196,48 +196,76 @@ private void insertTrivialData(ItemMeta meta) throws InvocationTargetException,
 	{
 		for (Method method : meta.getClass().getMethods())
 		{
-			Parameter[] parameters = method.getParameters();
-			if (parameters.length != 1 || !method.accessFlags().contains(AccessFlag.PUBLIC))
-			{
+			String methodName = method.getName();
+			if("wait".contentEquals(methodName)) continue;
+			if(methodName.startsWith("get")) continue;
+			if(methodName.startsWith("remove")) continue;
+
+			// This method cause error with trivial value. So we set convenient value
+			if(methodName.contentEquals("setAmplifier")){
+				method.invoke(meta, 4);
 				continue;
 			}
```

Comment:
Good catch here, did not think about this at all. I'm guessing this is what caused the unreliability issues when testing